SUMMARY.md

```diff
@@ -105,7 +105,8 @@
        * [Macro methods](syntax_and_semantics/macros/macro_methods.md)
        * [Hooks](syntax_and_semantics/macros/hooks.md)
        * [Fresh variables](syntax_and_semantics/macros/fresh_variables.md)
-   * [Attributes](syntax_and_semantics/attributes.md)
+   * [Annotations](syntax_and_semantics/annotations.md)
+       * [User Defined Annotations](syntax_and_semantics/annotations/user_defined_annotations.md)
```

Comment:
Can we keep this consistent with the other titles and not capitalize every word?
```suggestion
       * [User-defined annotations](syntax_and_semantics/annotations/user_defined_annotations.md)
```